src/micromamba/main.cpp

```diff
@@ -1272,26 +1328,14 @@ parse_file_options()
     }
 }
 
-
 void
 init_create_parser(CLI::App* subcom)
 {
-    subcom->add_option("specs", create_options.specs, "Specs to install into the new environment");
-    subcom->add_option("-p,--prefix", create_options.prefix, "Path to the Prefix");
-    subcom->add_option("-n,--name", create_options.name, "Name of the Prefix");
-    subcom->add_option("-f,--file", create_options.files, "File (yaml, explicit or plain)")
-        ->type_size(1)
-        ->allow_extra_args(false);
-    subcom->add_flag(
-        "--no-pin", create_options.no_pin, "Ignore pinned packages (from config or prefix file)");
+    init_install_global_parser(subcom);
 
     subcom->add_option(
         "--extra-safety-checks", create_options.extra_safety_checks, "Perform extra safety checks");
 
-    init_network_parser(subcom);
-    init_channel_parser(subcom);
-    init_global_parser(subcom);
-
```

Comment:
you tested that thsi can all be removed, right?

Reply:
all the content is now in `init_install_global_parser` to shared the code with `install` and `update`